### PR TITLE
Add button for manual on the left sidebar.

### PR DESCRIPTION
**Pull request by DominikBernhardt**
Also print the manual reference on the startpage in bold.

### CODE REVIEW

**Comment by alex-konovalov** [comment]
Also, on front page and in the repo description: "computer package" or "software package"?

**Comment by DominikBernhardt** [comment]
@alex-konovalov I don't have a particular opinion on this. Is "Software package" more standard? Then I'll choose that.

File: _layouts/default.html [hunk 1]
```diff
@@ -40,6 +40,8 @@ <h1 class="header">{{ site.data.package.name }}</h1>
           {% elsif site.data.package.github %}
           <li><a class="buttons github" href="{{ site.data.package.github }}">View On GitHub</a></li>
           {% endif %}
+          <li><a class="clean" href="{{ site.data.package.doc-html }}">Manual</a></li>
```
Comment (fingolfin) [inline _layouts/default.html]: That's not a button, though, it's a navigation link (nav link), no? Not meant to nitpick, I just was confused by the PR title ("button ?!?"), but the change itself makes sense.

File: stylesheets/styles.css [hunk 2]
```diff
@@ -308,6 +308,14 @@ a.github {
 	background: url(../images/octocat-small.png) no-repeat 1px;
 }
 
+a.clean {
+	-webkit-font-smoothing: antialiased;
+	font-weight: normal;
+	text-shadow: rgba(0, 0, 0, 0.4) 0 -1px 0;
+	padding: 2px 2px 2px 22px;
+  height: 30px;
```
Comment (fingolfin) [inline stylesheets/styles.css]: ... or maybe it's some kind of button after all?!? Can I see a demo rendering of this somewhere? Why is this CSS named "clean"? It seems to add a shadow, which doesn't sound like "cleaning" to me?!?

**Comment by DominikBernhardt** [comment]
@fingolfin The intend of this PR is to add a clickable object on the left sidebar that, when clicked on, redirects to the manual. I called it button because it is supposed to look like one. I did not want the same symbol like the download or GitHub one because that seemed confusing. I named the CSS `clean`, because it does not have any `.png` image next to it. I'm happy to rename the class. 
I don't know how I could make this available to you? I could send you a screenshot?

**Comment by gaehler** [comment]
One can see the rendered result of the PR [here](https://DominikBernhardt.github.io/carat).

**Comment by DominikBernhardt** [comment]
Ah thanks so much @gaehler !

**Comment by gaehler** [comment]
The GMP library should be added to the dependencies. Also, the HTML title still says it's a GAP package.

**Comment by DominikBernhardt** [comment]
Thanks @gaehler , this is done.

**Comment by DominikBernhardt** [comment]
@fingolfin Did you have a look at this? Again, I'm open to suggestions. So if there us a way you'd prefer this to look like then I'm happy to adapt the changes,

**Pull request merged**